app/src/androidTest/java/org/mozilla/fenix/ui/SmokeTest.kt

```diff
@@ -64,4 +66,192 @@ class SmokeTest {
+                }.openThreeDotMenu {
+                }.openEditBookmarks {
+                    clickdeleteBookmarkButton()
+                }.confirmBookmarkFolderDeletionAndGoBackToBrowser {
+                    verifySnackBarText("Deleted localhost")
+                }.openThreeDotMenu {
+                }.sharePage {
+                    // verifyShareAppsLayout()
+                    // verifyShareOverlay()
+                }.closeShareDialogReturnToPage {
+                }.openThreeDotMenu {
+                }.refreshPage {
+                    // verifyPageContent(defaultWebPage.content)
+                }.openTabDrawer {
+                    closeTabViaXButton(defaultWebPage.title)
+                }.openHomeScreen {
+                }
+            }
+        }
+    }
+
+    @Ignore
+    @Test
+    fun verifyPageMainMenuItemsListLandscapeMode() {
+        val defaultWebPage = TestAssetHelper.getGenericAsset(mockWebServer, 1)
+        activityTestRule.getActivity().setRequestedOrientation(ActivityInfo.SCREEN_ORIENTATION_LANDSCAPE)
+
+        homeScreen {
+            mDevice.waitForIdle(waitingTimeShort)
+            togglePrivateBrowsingModeOnOff()
+            navigationToolbar {
+            }.enterURLAndEnterToBrowser(defaultWebPage.url) {
+                activityTestRule.getActivity().setRequestedOrientation(ActivityInfo.SCREEN_ORIENTATION_LANDSCAPE)
+            }.openThreeDotMenu {
+                verifyThreeDotMainMenuItems()
+            }.openHistory {
+                // verifyTestPageUrl(defaultWebPage.url)
+            }.goBackToBrowser {
+            }.openThreeDotMenu {
+            }.openBookmarks {
+                verifyBookmarksMenuView()
+                verifyEmptyBookmarksList()
+            }.goBackToBrowser {
+            }.openThreeDotMenu {
+            }.openSettings {
+                verifySettingsView()
+            }.goBackToBrowser {
+            }.openThreeDotMenu {
+            }.openFindInPage {
+                verifyFindInPageSearchBarItems()
+            }.closeFindInPage {
+            }.openThreeDotMenu {
+            }.addToFirefoxHome {
+                verifySnackBarText("Added to top sites!")
+            }.openTabDrawer {
+            }.openTab(defaultWebPage.title) {
+            }.openThreeDotMenu {
+            }.openAddToHomeScreen {
+                verifyShortcutNameField(defaultWebPage.title)
+                clickAddShortcutButton()
+                clickAddAutomaticallyButton()
+                verifyShortcutIcon()
+            }.openHomeScreenShortcut(defaultWebPage.title) {
+            }.openThreeDotMenu {
+            }.bookmarkPage {
+                verifySnackBarText("Bookmark saved!")
+            }.openThreeDotMenu {
+                verifyEditBookmarkButton()
+            }.openEditBookmarks {
+                clickdeleteBookmarkButton()
+            }.confirmBookmarkFolderDeletionAndGoBackToBrowser {
+                verifySnackBarText("Deleted localhost")

```

Comment:
Same here, all these steps look out of scope for this test and they complicate and add a lot of extra code for now... I would say this is covered by other tests too, @sv-ohorvath do you know? Thanks!

Reply:
I added this because, I could continue with the below  in private mode as well.

So, added a bookmark and removed it and when that goes with private mode it directly adds a bookmark as the page hasn't been bookmarked already

Reply:
If the tests are separated for regular/private mode then there may be a way to have it working?

Reply:
Okay